v3/docs/TUTORIAL.md

```diff
@@ -644,7 +644,7 @@ from uasyncio import Event
 async def waiter(event):
     print('Waiting for event')
     await event.wait()  # Pause here until event is set
-    print('Waiter got event.')
+    print('Waiter got event')
```

Comment:
What is wrong with the '.' here?

Reply:
This is about consistency. Most prints don't have a period.